test/wdio/slot-fallback-with-textnode/cmp.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { h } from '@stencil/core';
+import { render, waitForChanges } from '@wdio/browser-runner/stencil';
+
+describe('slot-fallback-with-textnode', function () {
+  beforeEach(() => {
+    render({
+      template: () => <my-component></my-component>,
+    });
+  });
+
+  it('should hide fallback content when provided slot is text node', async () => {
+    await $('#toggle-button').click();
```

Comment:
Let's validate the initial state:
```suggestion
    await expect($('.container')).toHaveText(expect.stringContaining('DEFAULT'))
    await $('#toggle-button').click();
```

Reply:
done